tests/foreman/virtwho/test_ui_virtwho.py

```diff
@@ -134,3 +128,220 @@ def test_positive_deploy_configure_by_script(session, form_data):
+            session.virtwho_configure.edit(name, {'interval': option})
+            deploy_configure_by_command(config_command)
+            assert get_configure_option('VIRTWHO_INTERVAL', VIRTWHO_SYSCONFIG) == value
+        session.virtwho_configure.delete(name)
+        assert not session.virtwho_configure.search(name)
+
+
+@tier2
+def test_positive_hypervisor_id_option(session, form_data):
+    """ Verify Hypervisor ID dropdown options.
+
+    :id: dd490f5f-8371-4be6-81d5-a02bbff43de8
+
+    :expectedresults:
+        hypervisor_id can be changed in virt-who-config-{}.conf if the
+        dropdown option is selected to uuid/hwuuid/hostname.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        config_file = get_configure_file(config_id)
+        values = ['uuid', 'hwuuid', 'hostname']
+        for value in values:
+            session.virtwho_configure.edit(name, {'hypervisor_id': value})
+            deploy_configure_by_command(config_command)
+            assert get_configure_option('hypervisor_id', config_file) == value
+        session.virtwho_configure.delete(name)
+        assert not session.virtwho_configure.search(name)
+
+
+@skip_if_bug_open('bugzilla', 1735670)
+@tier2
+def test_positive_filtering_option(session, form_data):
+    """ Verify Filtering dropdown options.
+
+    :id: 107c5644-cbf4-48b6-8975-718b661177b0
+
+    :expectedresults:
+        1. if filtering is selected to Whitelist, 'Filter hosts' can be set.
+        2. if filtering is selected to Blacklist, 'Exclude hosts' can be set.
+
+    :CaseLevel: Integration
+
+    :CaseImportance: Medium
+    """
+    name = gen_string('alpha')
+    form_data['name'] = name
+    with session:
+        session.virtwho_configure.create(form_data)
+        config_id = get_configure_id(name)
+        config_command = get_configure_command(config_id)
+        config_file = get_configure_file(config_id)
+        hypervisor_type = form_data['hypervisor_type']
+        regex = '.*redhat.com'
+        whitelist = {
+            'filtering': 'Whitelist',
+            'filtering_content.filter_hosts': regex,
+        }
+        blacklist = {
+            'filtering': 'Blacklist',
+            'filtering_content.exclude_hosts': regex,
+        }
+        if hypervisor_type == 'esx':
+            whitelist['filtering_content.filter_host_parents'] = regex
+            blacklist['filtering_content.exclude_host_parents'] = regex
+        # Update Whitelist and check the result
+        session.virtwho_configure.edit(name, whitelist)
+        deploy_configure_by_command(config_command)
+        assert regex == get_configure_option('filter_hosts', config_file)
```

Comment:
why are you not validating the `assert` based on UI search ? and same for all below `assert` where used in same way ..

Reply:
as the above, the final goal of virt-who configure is help to change the /etc/sysconfig/virt-who or the configure file in /etc/virt-who.d.

Reply:
same as above ..

Reply:
I have a little afraid it will spend more time to run the case if validating all the options' value, but according to the latest test result with the new commit, it's acceptable:
` 7 passed, 1 skipped in 2829.53 seconds`